scripts/discovery/analysis/pass5/pass5_elicit.py

```
@@ -0,0 +1,708 @@
+    model: str,
+    system_prompt: str,
+    out_path: Path,
+    resume: bool,
+    progress_every: int,
+) -> dict:
+    """Generic phase runner: replicate-interleaved schedule, jsonl-append.
+
+    For replicate r in 0..replicates-1, iterate inputs in given order.
+    Pass 4-style resume: skip already-committed call_idx values.
+    """
+    next_idx, committed = load_resume_state(out_path) if resume else (1, set())
```

Ruff (0.15.12)

[warning] Unpacked variable `next_idx` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)